### ISSUE TITLE
Option to verify yourself as a contact

### ISSUE THREAD

**Issue by tiffrobo** [zom/Zom-iOS#525]
T has the option to verify herself as a contact.
Second time this has happened in build 117. 

![img_3861](https://user-images.githubusercontent.com/30634981/35525737-01b7d1fa-04eb-11e8-8c96-443f25da52f5.PNG)

Device: iPhone 7
OS: 11.2.2
App build: 117

**Comment by N-Pex** [comment]
@chrisballinger I don't think this is a bug, it just means that you have signed in on another device and is a type of "is this really you" thing? I think there is a ticket for it, or at least it is related to the "Something has changed" ticket.

**Comment by tiffrobo** [comment]
@N-Pex @chrisballinger I haven't signed into my account on another device. However, my account does sign in and sign out occasionally.
